.pylintrc

```diff
@@ -0,0 +1,12 @@
+# pylint config
+[FORMAT]
+good-names=m,p,ok
+[MESSAGES CONTROL]
+disable=fixme,
+        line-too-long,
+        too-many-locals,
+        too-many-statements,
+        too-many-branches,
+        missing-function-docstring,
+        missing-class-docstring,
+        consider-using-f-string
```

Comment:
Github complains about missing `newline`, since this file will not get linted (I guess?) this shouldn't matter?

Reply:
Done.